drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -1918,6 +1918,7 @@ static int usb_init(const struct device *arg)
 			   K_KERNEL_STACK_SIZEOF(usbd_work_queue_stack),
 			   CONFIG_SYSTEM_WORKQUEUE_PRIORITY, NULL);
 
+	(void)k_thread_name_set(&usbd_work_queue.thread, "usbd_workq");
```

Comment:
k_thread_name_set() is called without casting the result in other places, why would the cast be needed here?

Reply:
See 'drivers\modem\gsm_ppp.c' as an example where it's casted. Can easily remove the cast.

Reply:
Removed the cast

Reply:
Please remove the `(void)`, I see most callers don't use it.